### PR TITLE
Fix publications queries

### PR DESCRIPTION
**Pull request by csoule-air**

### CODE REVIEW

**Comment by csoule-air** [comment]
==================================== test session starts =====================================
platform darwin -- Python 3.11.4, pytest-7.3.1, pluggy-1.2.0
rootdir: /Users/csoule/repos/es-data-load
collected 22 items                                                                           

tests/test_LoadJob.py ....                                                             [ 18%]
tests/test_MySQLSource.py ....                                                         [ 36%]
tests/test_es.py ..                                                                    [ 45%]
tests/mappings/test_mappings.py .s.                                                    [ 59%]
tests/schema/test_schema.py ....                                                       [ 77%]
tests/utilities/test_utils.py .....                                                    [100%]

====================================== warnings summary ======================================
tests/test_LoadJob.py: 3 warnings
tests/test_es.py: 2 warnings
tests/schema/test_schema.py: 3 warnings
tests/utilities/test_utils.py: 2 warnings
  /Users/csoule/repos/es-data-load/src/es_data_load/es.py:31: DeprecationWarning: The 'timeout' parameter is deprecated in favor of 'request_timeout'
    self.es = Elasticsearch(

tests/test_LoadJob.py: 6 warnings
tests/schema/test_schema.py: 31 warnings
  /Users/csoule/repos/es-data-load/src/es_data_load/pv/schemas/PVSchemaManager.py:54: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    with pkg_resources.path(

tests/test_LoadJob.py: 6 warnings
tests/schema/test_schema.py: 9 warnings
tests/utilities/test_utils.py: 8 warnings
  /Users/csoule/repos/es-data-load/src/es_data_load/lib/utilities.py:39: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    es.indices.create(index=index_name, body={"mappings": {"properties": fields}})

tests/test_LoadJob.py::test_process_load_operation
tests/test_es.py::test_bulk_load_es_documents
  /Users/csoule/repos/es-data-load/tests/test_es.py:16: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/test_LoadJob.py::test_process_load_operation
tests/test_es.py::test_bulk_load_es_documents
  /Users/csoule/repos/es-data-load/tests/test_es.py:22: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/test_LoadJob.py::test_process_csv_load_operation
  /Users/csoule/repos/es-data-load/tests/test_LoadJob.py:128: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/test_LoadJob.py::test_process_csv_load_operation
  /Users/csoule/repos/es-data-load/tests/test_LoadJob.py:134: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/mappings/test_mappings.py: 21 warnings
  /Users/csoule/repos/es-data-load/tests/mappings/test_mappings.py:24: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    pkg_resources.path("es_data_load.pv.mappings.production.granted", fl)

tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
  /Users/csoule/repos/es-data-load/src/es_data_load/pv/mappings/PVMappingsManager.py:73: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    pkg_resources.path("es_data_load.pv.mappings.production.granted", fl)

tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
  /Users/csoule/repos/es-data-load/src/es_data_load/pv/mappings/PVMappingsManager.py:77: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    pkg_resources.path("es_data_load.pv.mappings.production.pregrant", fl)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 21 passed, 1 skipped, 104 warnings in 31.24s ========================

File: tests/schema/test_schema.py [hunk 1]
```diff
@@ -61,7 +62,8 @@ def test_go_live(search):
     live_flag_name = "_test_live"
     suffix_name = "_test"
     sm = PVSchemaManager.load_default_pv_schema(
-        suffix=suffix_name, search_wrapper=search, granted_schema_files=random_choices
+        suffix=suffix_name, search_wrapper=search, granted_schema_files=random_choices,
+        pregrant_schema_files=[]
```
Comment (sarvothaman) [inline tests/schema/test_schema.py:66]: Could we include a pregrant file in the test as well?

File: tests/mappings/test_mappings.py [hunk 2]
```diff
@@ -51,8 +51,9 @@ def test_pv_load_configuration():
     pv_configuration = PVLoadConfiguration.load_default_pv_configuration(
         suffix="_test",
         granted_files=random.sample(AVAILABLE_MAPPING_FILES["granted"], k=5),
+        pregrant_files=random.sample(AVAILABLE_MAPPING_FILES["pregrant"], k=2),
```
Comment (sarvothaman) [inline tests/mappings/test_mappings.py:54]: Do we have only 2 files in pregrant? If so may be it makes sense to just list them here rather than sampling?

**Review by sarvothaman — CHANGES_REQUESTED** [review]

**Comment by csoule-air** [comment]
==================================== test session starts =====================================
platform darwin -- Python 3.11.4, pytest-7.3.1, pluggy-1.2.0
rootdir: /Users/csoule/repos/es-data-load
collected 22 items                                                                           

tests/test_LoadJob.py ....                                                             [ 18%]
tests/test_MySQLSource.py ....                                                         [ 36%]
tests/test_es.py ..                                                                    [ 45%]
tests/mappings/test_mappings.py .s.                                                    [ 59%]
tests/schema/test_schema.py ....                                                       [ 77%]
tests/utilities/test_utils.py .....                                                    [100%]

====================================== warnings summary ======================================
tests/test_LoadJob.py: 3 warnings
tests/test_es.py: 2 warnings
tests/schema/test_schema.py: 3 warnings
tests/utilities/test_utils.py: 2 warnings
  /Users/csoule/repos/es-data-load/src/es_data_load/es.py:31: DeprecationWarning: The 'timeout' parameter is deprecated in favor of 'request_timeout'
    self.es = Elasticsearch(

tests/test_LoadJob.py: 6 warnings
tests/schema/test_schema.py: 33 warnings
  /Users/csoule/repos/es-data-load/src/es_data_load/pv/schemas/PVSchemaManager.py:54: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    with pkg_resources.path(

tests/test_LoadJob.py: 6 warnings
tests/schema/test_schema.py: 11 warnings
tests/utilities/test_utils.py: 8 warnings
  /Users/csoule/repos/es-data-load/src/es_data_load/lib/utilities.py:39: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    es.indices.create(index=index_name, body={"mappings": {"properties": fields}})

tests/test_LoadJob.py::test_process_load_operation
tests/test_es.py::test_bulk_load_es_documents
  /Users/csoule/repos/es-data-load/tests/test_es.py:16: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/test_LoadJob.py::test_process_load_operation
tests/test_es.py::test_bulk_load_es_documents
  /Users/csoule/repos/es-data-load/tests/test_es.py:22: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/test_LoadJob.py::test_process_csv_load_operation
  /Users/csoule/repos/es-data-load/tests/test_LoadJob.py:128: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/test_LoadJob.py::test_process_csv_load_operation
  /Users/csoule/repos/es-data-load/tests/test_LoadJob.py:134: DeprecationWarning: The 'body' parameter is deprecated and will be removed in a future version. Instead use individual parameters.
    hits = search.es.search(

tests/mappings/test_mappings.py: 21 warnings
  /Users/csoule/repos/es-data-load/tests/mappings/test_mappings.py:24: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    pkg_resources.path("es_data_load.pv.mappings.production.granted", fl)

tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
  /Users/csoule/repos/es-data-load/src/es_data_load/pv/mappings/PVMappingsManager.py:73: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    pkg_resources.path("es_data_load.pv.mappings.production.granted", fl)

tests/mappings/test_mappings.py::test_pv_load_configuration
tests/mappings/test_mappings.py::test_pv_load_configuration
  /Users/csoule/repos/es-data-load/src/es_data_load/pv/mappings/PVMappingsManager.py:77: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    pkg_resources.path("es_data_load.pv.mappings.production.pregrant", fl)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
======================== 21 passed, 1 skipped, 108 warnings in 34.57s ========================

**Review by sarvothaman — APPROVED** [review]

**Pull request merged**